__tests__/app.test.js

```diff
@@ -153,3 +153,56 @@ describe("5. GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("6. GET /api/reviews/:review_id/comments", () => {
+  test("GET-200 returns an array of comments object for the passed review_id and check that the passed review_id is returned with the object", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: 2,
+          });
+        });
+      });
+  });
+  test("GET-200 return the comments object sorted in a descending order", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("GET- 404 sends comment does not exist error message when given a valid but non-existent id in the comments table", () => {
```

Comment:
I'd argue against this - if a review exists but has no comments yet, thinking from a user's perspective we wouldn't normally expect to see an error (for example if a facebook post has no comments - you don't get a 404). I'd refactor to test that if there are zero comments, just expect a 200 and an empty array instead!